src/hotspot/share/oops/compressedKlass.cpp

```diff
@@ -32,9 +32,30 @@
 address CompressedKlassPointers::_base = nullptr;
 int CompressedKlassPointers::_shift = 0;
 size_t CompressedKlassPointers::_range = 0;
+uint64_t CompressedKlassPointers::_combo = 0;
 
 #ifdef _LP64
 
+void CompressedKlassPointers::set_base_and_shift(address thebase, int theshift) {
+  assert(UseCompressedClassPointers, "Why are we here?");
```

Comment:
All other Hotspot asserts have this form:

```suggestion
  assert(UseCompressedClassPointers, "only for compressed klass code");
```